docs/Benchmarks.md

```diff
@@ -0,0 +1,94 @@
+# Benchmarks
+
+This document explains how to benchmark ASP.NET Core changes. It is intended for Microsoft employees.
+It describes how to trigger benchmarks using the Crank CLI and the Pull-Request bot.

```

Comment:
Consider a section here describing what PR's we imagine should use such runs? Essentially -- what kinds of changes tend to lead to perf regressions (that would be caught) and what would not benefit. This may also depend on how heavily the machines are subscribed -- that could feed into guidance here.

(If/when we set up a bot to post an encouragement/instructions into PR's that would most benefit from a perf run, perhaps by looking at files changed, that will obviously also help. Note that the labeler bot, eg., already looks at files changed in a PR.)

Reply:
I'd rather keep this document as succint as possible for devs who want to run a benchmark but don't remember how to. My personal opinion for which PRs should use such run is only "for external contributor PRs", and have .NET devs use locally built branches instead. I mentioned that in the dedicated paragraph.